web/beacon-app/src/components/auth/LandingFooter/LandingFooter.tsx

```diff
@@ -20,11 +20,24 @@ function LandingFooter() {
                   <a href="https://rotational.io/ensign">Ensign</a>
                 </li>
                 <li>
-                  <a href="https://ensign.rotational.dev/getting-started/">Documentation</a>
+                  <a
+                    href="https://ensign.rotational.dev/getting-started/"
```

Comment:
Don’t you think it is better to put this link in the “EXTERNAL_LINKS” object to avoid modifications in multiple places later?

Reply:
Most of the footer links were added before that object was created, so I've moved all of them to `EXTERNAL_LINKS`.